client/packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -30,12 +30,14 @@ const useDisableOutboundRows = (rows?: OutboundRowFragment[]) => {
 export const OutboundShipmentListViewComponent: FC = () => {
   const { mutate: onUpdate } = useOutbound.document.update();
   const t = useTranslation('distribution');
+  const { updateSortQuery, updatePaginationQuery, filter } =
+    useHandleUrlQueryParams('otherPartyName');
```

Comment:
I know that it's more verbose.. but if you do include a sort param, then passing in an object would be easier to read:

```
useHandleUrlQueryParams({ filterKey: 'otherPartyName', sortKey: 'otherPartyName' });
```

🤷

Reply:
Just saw this -- but that's exactly what I've just done 👍

Reply:
All suggested changes to this point: bde3e6052